### ISSUE TITLE
Only the original thread that created a view hierarchy can touch its views.

### ISSUE THREAD

**Issue by ivanhuay** [software-mansion/react-native-gesture-handler#971]
I'm getting this error 

`Only the original thread that created a view hierarchy can touch its views.`

This happens randomly when i use `TouchableWithoutFeedback` component.

![image](https://user-images.githubusercontent.com/5782286/74558715-8ae5b580-4f41-11ea-8d14-27c7e46e0411.png)

Do you have any idea about that? 

i'm using:
```
    "react-native": "0.60.5",
    "react-native-gesture-handler": "^1.6.0",
```
device:
`nexus 4  -  android 6`

Sample code: https://github.com/ivanhuay/gesture-test

I think this happens sometimes when you click a component and you are using some conditional rendering. So maybe some events are firing when the component is out of render.

**Comment by kenMarquez** [comment]
@ivanhuay  Did you solve your problem?

**Comment by ivanhuay** [comment]
@kenMarquez not really I just made a quick fix forcing the onPressOut. I changed the size and opacity before "destroy" the components so the user can't tap when the conditional render changes.

**Comment by jakub-gonet** [comment]
Resolved in 1.8.0.
